manuscript/210_How_are_objects_composed.md

```diff
@@ -1169,7 +1169,7 @@ Putting it all together, factories allow giving names to some specific object co
 
 #### Factories help eliminate redundancy
 
-Redundancy in code means that at least two things need to change for the same reason in the same way[^essentialskills]. Usually it is understood as code duplication, but actually, "conceptual duplication" is a better term. For example, the following two methods are not redundant, even though the code seems duplicated (by the way, the following is not an example of good code, just a simple illustration):
+Redundancy in code means that at least two things need to change for the same reason in the same way[^essentialskills]. Usually it is understood as code duplication, but "conceptual duplication" is a better term. For example, the following two methods are not redundant, even though the code seems duplicated (by the way, the following is not an example of good code, just a simple illustration):
```

Comment:
How about:
"but "conceptual duplication" is a better term"
->
"but I consider "conceptual duplication" a better term"
?